docs/how-to/create-initial-data-migrations.md

```diff
@@ -0,0 +1,78 @@
+# Create initial data scripts
+
+## Overview
+
+The goal is to convert our initial data into scripts that can be loaded into the database when the backend is set up for the first time.
+
+These are the steps:
+
+   1. Export the data into JSON
+   1. Generate a python script from the JSON data
+
+### Prerequisites
+
+The initial data exists in a Google spreadsheet, such as [this one for People Depot][pd-data-spreadsheet]. There should be individual sheets named after the model names the data correspond to, such as `SOC Major - Data`. The sheet name is useful for us to identify the model it corresponds to.
+
+The sheet should be formatted like so:
+
+- the first row contains the names of the field names in the model. The names must be exactly the same
+- rows 2 to n are the initial data for the model we want to turn into a script.
+
+## Convert the data into JSON
+
+   1. Export the data from the Google [spreadsheet][pd-data-spreadsheet]
+      1. Find the sheet in the document containing the data to export. Let's use the `SOC Major - Data` data as our example. It's
+      1. Make sure that the first row (column names) is frozen. Otherwise, freeze it by selecting the first row in the sheet, then Menu > View > Freeze > Up to row 1
+      1. Export to JSON. Menu > Export JSON > Export JSON for this sheet
+   1. Save the JSON into a file
```

Comment:
The current steps do not indicate there is a dialog box, it reads better with sub steps, the last step is a note.  If these changes are made for lines 27-30 it would look like this:
   1. Export to JSON. Menu > Export JSON > Export JSON for this sheet. A dialog box with JSON will appear.  
      1. Select all the JSON and copy into the clipboard.
      ==> Add this line, 
   1. Save the JSON as [ModelNameInPascalCase]_export.json under app/core/initial_data/


      ==> Change to add NOTE:.  It is not a step.
      NOTE: The Pascal case is important in the next step to generate a python script to insert the data. It must match the model's class name for this to work.